site/pages/about-us/__snapshots__/test.js.snap

```diff
@@ -1073,12 +1087,26 @@ exports[`site/pages/about-us renders correctly 1`] = `
         tweets={
           Array [
             Object {
-              "created": "1988-02-11",
+              "created": "1988-02-12",
```

Comment:
You removed the reference to my birthday! 😭

Reply:
Haha, oh sorry